src/components/tiltak/LagretTiltak.tsx

```diff
@@ -87,22 +87,17 @@ export const LagretTiltak = ({
 
   const tiltakId = tiltak.tiltakId;
 
-  const manglerVurderingFraLeder =
-    tiltak &&
-    !tiltak.gjennomfoering &&
-    !tiltak.beskrivelseIkkeAktuelt &&
-    arbeidstakerFnr === tiltak.opprettetAv?.fnr &&
-    tiltak.sistEndretAv.fnr === arbeidstakerFnr;
-
-  const kanSletteTiltak = () => {
+  function isTiltakCreatedByInnloggetRole() {
     if (isAudienceSykmeldt) {
-      return arbeidstakerFnr === tiltak.opprettetAv?.fnr;
-    } else {
-      return (
-        !manglerVurderingFraLeder && innloggetFnr === tiltak.opprettetAv?.fnr
-      );
+      return innloggetFnr === tiltak.opprettetAv.fnr;
     }
-  };
+
+    return arbeidstakerFnr !== tiltak.opprettetAv.fnr;
+  }
+
+  function isVudert() {
```

Comment:
isVurdert

Reply:
Ellers, er det riktig definisjon at den må ha "beskrivelseIkkeAktuelt" for at den skal være vurdert? Hva om den  vurderes som kan gjennomføres, da er det vel ingen beskrivelseIkkeAktuelt

Reply:
Jeg antok at hvis tiltak kan gjennomføres blir dette feltet satt:`tiltak.gjennomfoering`

Reply:
Ja kan hende det blir satt til tom streng eller noe, men hørtes litt rart å sjekke på den dersom man velger en annen gjennomføring? Ser den er de før da. 

Btw isVudert -> isVurdert :D